src/main/java/com/google/devtools/build/lib/worker/WorkerKey.java

```diff
@@ -110,6 +110,15 @@ public boolean getProxied() {
     return proxied;
   }
 
+  /** Getter function for worker type name. */
```

Comment:
nit: Should be something like `Returns a user-friendly name for this worker type.`

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
Done.